iml-systemd/src/lib.rs

```diff
@@ -2,7 +2,9 @@
 // Use of this source code is governed by a MIT-style
 // license that can be found in the LICENSE file.
 
-use crate::agent_error::{ImlAgentError, RequiredError};
+mod systemd_error;
+
+use crate::systemd_error::{RequiredError, SystemdError};
```

Comment:
```suggestion
use crate::systemd_error::RequiredError;
pub use crate::systemd_error::SystemdError;
```

We should re-export `SystemdError` so it can be easily imported in other crates.

Reply:
Makes sense 👍